packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -255,14 +298,37 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
           this.forceUpdate();
         }
 
+        document.addEventListener('keydown', this._onKeyDown, true);
+
         // We dont want to the user to immediatley trigger the coachmark when it's opened
         this._async.setTimeout(() => {
           this._addProximityHandler(this.props.mouseProximityOffset);
         }, this.props.delayBeforeMouseOpen!);
+
+        // SetTimeout is hacky solution.  ARIA text doesn't read aloud unless it is appended or changed after the component
+        // is mounted
+        if (this.props.ariaAlertText) {
+          this._async.setTimeout(() => {
+            if (this._ariaAlertContainer.current && this.props.ariaAlertText) {
+              this._ariaAlertContainer.current.innerText = this.props.ariaAlertText;
+            }
+          }, 2000);
+        }
       }
     );
   }
 
+  public componentWillUnmount(): void {
+    document.removeEventListener('keydown', this._onKeyDown);
+  }
+
+  private _onKeyDown = (e: KeyboardEvent): void => {
+    // Open coachmark if user presses ALT + C (arbitrary keypress for now, will change later)
```

Comment:
Can you elaborate here? Not entirely sure what your ask is.  Are you asking if we can/should customize the shortcut to open Coachmark?  @cschlechty , can you chime in here if you have a chance please?

Reply:
No, I'm asking how a user using Narrator actually enters and highlights the Coachmark using keyboard navigation. Tabbing seems to go past it. I couldn't find a navigation standard way of highlighting a non-modal dialog. Alt+C highlights it, but I'm curious how a user using assistive technology would know this.. is it standardized anywhere?

Reply:
It's not a stand control or pattern. We're inventing it, and pushing its usage instructions via embedded usage string. @leddie24  be sure this is included in the examples.

Reply:
Is that embedded usage string in this PR? I'm assuming it informs users about hitting ALT+C. I couldn't find it.. thanks

Reply:
So it will only be set from outside the control? The default selection behavior is embedded in the control, so I'm wondering if the default notification on usage should be as well. (Similar to how "A coachmark has appeared" is.)

Reply:
"A coachmark has appeared"  and the usage instructions need to be localize-able, so they'll need to be passed in.

Reply:
I understand that, but some are defaulted to English within the component and others are not. I feel like the navigation info is probably the most important one to be present.